cw_mweb/lib/cw_mweb.dart

```diff
@@ -197,4 +207,18 @@ class CwMweb {
     }
     return null;
   }
+
+  static Future<BroadcastResponse> broadcast(BroadcastRequest request) async {
+    log("mweb.broadcast() called");
+    try {
+      _rpcClient = await stub();
+      return await _rpcClient!.broadcast(request, options: CallOptions(timeout: TIMEOUT_DURATION));
+    } on GrpcError catch (e) {
+      log('Caught grpc error: ${e.message}');
+      rethrow;
```

Comment:
let's make it like this so we know if it's coming from here, what do you think?
```suggestion
      throw "error from broadcast mweb: $e";
```